config/locales/simple_form.en.yml

```diff
@@ -4,7 +4,7 @@ en:
     "no": 'No'
     required:
       text: 'required'
-      mark: '*'
+      mark: ''
```

Comment:
FYI - this is to remove the asterisk for mandatory fields because it ruins the style of the modal :(